### DISCUSSION TITLE
Bare Metal option for worker/agent node?

### DISCUSSION THREAD

**Discussion by supernovae** [mysticaltech/terraform-hcloud-kube-hetzner#893] [Q&A]
Has anyone done any scripting to be able to add hetzner bare metal servers to a cluster? Is there a straightforward combustion i can run if i provision a bare metal machine on the right network to have it join? I'd like to add a bunch of Ryzen 3600s for some workloads that enjoy dedicated CPUs (and costs a bit less than cloud dedicated)

any tips would be great.

**Comment by mysticaltech** [comment]
@supernovae Some folks have succeeded before in this project, search for it, however it does require some custom work. If one day that becomes more feasible, we will definitely add official support.
